drivers/timers/rtc.c

```diff
@@ -844,11 +844,11 @@ int rtc_initialize(int minor, FAR struct rtc_lowerhalf_s *lower)
   upper->unlinked = false;  /* Driver is not  unlinked */
 #endif
 
-  /* Create the driver name.  There is space for the a minor number up to  6
+  /* Create the driver name.  There is space for the a minor number up to 10
    * characters
    */
 
-  snprintf(devpath, 16, "/dev/rtc%d", minor);
+  snprintf(devpath, 20, "/dev/rtc%d", minor);
```

Comment:
Please use `sizeof(devpath)` instead of hard-code value 20

Reply:
Done.